sandbox/icons.html.example

```diff
@@ -33,6 +33,38 @@
     <li><span class="vjs-icon-circle"></span> <code>.vjs-icon-circle</code></li>
     <li><span class="vjs-icon-circle-outline"></span> <code>.vjs-icon-circle-outline</code></li>
     <li><span class="vjs-icon-circle-inner-circle"></span> <code>.vjs-icon-circle-inner-circle</code></li>
+
+          <li><span class="play"></span> <code>.vjs-icon-play</code></li>
+          <li><span class="play-circle"></span> <code>.vjs-icon-play</code></li>
```

Comment:
Because it's documentation and it's related to one of the guides that was edited.